app/controllers/TransferConfirmationController.scala

```diff
@@ -0,0 +1,5 @@
+package controllers
+
+class TransferConfirmationController {
```

Comment:
Does this controlled need the security components?

Reply:
Yes, but this is just a placeholder controller since I'm not building this page (at least for this story), I just have to redirect the transfer summary page to it; I guess I could delete it?

Reply:
I would remove it, if its not needed for this task.

Reply:
Ok